postprocessing/comstockpostproc/resources/comstock_column_definitions.csv

```diff
@@ -647,297 +647,285 @@ results.csv,com_stock_sensitivity_reports.com_report_smallest_space_m_2,out.para
-results.csv,emissions_reporting.annual_electricity_ghg_emissions_egrid_2018_state_kg,out.emissions.electricity.egrid_2018_state,TRUE,FALSE,float,co2e_kg,co2e_kg,"annual greenhouse gas emissions from electricity use, using egrid 2018 state emissions intensity"
-results.csv,emissions_reporting.annual_electricity_ghg_emissions_egrid_2018_subregion_kg,out.emissions.electricity.egrid_2018_subregion,TRUE,FALSE,float,co2e_kg,co2e_kg,"annual greenhouse gas emissions from electricity use, using egrid 2018 subregion emissions intensity"
-results.csv,emissions_reporting.annual_electricity_ghg_emissions_egrid_2019_state_kg,out.emissions.electricity.egrid_2019_state,TRUE,FALSE,float,co2e_kg,co2e_kg,"annual greenhouse gas emissions from electricity use, using egrid 2019 state emissions intensity"
-results.csv,emissions_reporting.annual_electricity_ghg_emissions_egrid_2019_subregion_kg,out.emissions.electricity.egrid_2019_subregion,TRUE,FALSE,float,co2e_kg,co2e_kg,"annual greenhouse gas emissions from electricity use, using egrid 2019 subregion emissions intensity"
-results.csv,emissions_reporting.annual_electricity_ghg_emissions_egrid_2020_state_kg,out.emissions.electricity.egrid_2020_state,TRUE,FALSE,float,co2e_kg,co2e_kg,"annual greenhouse gas emissions from electricity use, using egrid 2020 state emissions intensity"
-results.csv,emissions_reporting.annual_electricity_ghg_emissions_egrid_2020_subregion_kg,out.emissions.electricity.egrid_2020_subregion,TRUE,FALSE,float,co2e_kg,co2e_kg,"annual greenhouse gas emissions from electricity use, using egrid 2020 subregion emissions intensity"
-results.csv,emissions_reporting.annual_electricity_ghg_emissions_egrid_2021_state_kg,out.emissions.electricity.egrid_2021_state,TRUE,FALSE,float,co2e_kg,co2e_kg,"annual greenhouse gas emissions from electricity use, using egrid 2021 state emissions intensity"
-results.csv,emissions_reporting.annual_electricity_ghg_emissions_egrid_2021_subregion_kg,out.emissions.electricity.egrid_2021_subregion,TRUE,FALSE,float,co2e_kg,co2e_kg,"annual greenhouse gas emissions from electricity use, using egrid 2021 subregion emissions intensity"
-results.csv,emissions_reporting.winter_daily_average_electricity_ghg_emissions_egrid_2021_state_kg,out.emissions.electricity.winter_daily_average.egrid_2021_state,TRUE,FALSE,float,co2e_kg,co2e_kg,"winter daily average greenhouse gas emissions from electricity use, using egrid 2021 state intensity"
-results.csv,emissions_reporting.winter_daily_average_electricity_ghg_emissions_egrid_2021_subregion_kg,out.emissions.electricity.winter_daily_average.egrid_2021_subregion,TRUE,FALSE,float,co2e_kg,co2e_kg,"winter daily average greenhouse gas emissions from electricity use, using egrid 2021 subregion intensity"
```

Comment:
@eringold I noticed you deleted these "daily_average" emissions entries. They were added specifically for the emission objective lighting measure recently. Could we add them back?

Reply:
@JieXiong9119 this was my mistake, I missed those in the conflict resolution in this file. I can make a PR can fix tomorrow or Thurs.

Reply:
Thanks! FYI, I believe there are 15 rows with substring `daily_average_electricity_ghg_emission`.